src/OrchardCore/OrchardCore.FileStorage.AzureBlob/BlobFileStore.cs

```diff
@@ -145,7 +145,7 @@ await foreach (var blob in page)
                 }
                 else
                 {
-                    var itemName = Path.GetFileName(WebUtility.UrlDecode(blob.Blob.Name)).Trim('/');
+                    var itemName = Path.GetFileName(blob.Blob.Name).Trim('/');
```

Comment:
The decode code is old from the previous blob library from Microsoft.

It maybe that it is no longer delivered by the azure library encoded, so a decode doesn't do anything useful.

Reply:
the `decode` can damage file path. `aaa+++.jpg` -> `aaa   .jpg`
remove `decode` can resolve it if use Kestrel Server directly.
but need to set `allowDoubleEscaping="true"` in web.config if use IIS Server.